modules/custom/az_core/css/az-bef-sidebar.css

```diff
@@ -0,0 +1,119 @@
+.az-bef-vertical ul {
+  list-style: none;
```

Comment:
This can be replaced with https://digital.arizona.edu/arizona-bootstrap/docs/2.0/content/typography/#unstyled